sites/search/search-en.html

```diff
@@ -0,0 +1,17 @@
+---
+{
+	"altLangPage": "search-fr.html",
+	"dateModified": "2023-07-17",
+	"description": "Documentation on how to use search bar.",
+	"language": "en",
+	"title": "GC Site Search Bar example"
+}
+---
+<div class="wb-prettify all-pre hide"></div>
+
+<p>To see a practical implementation of the GC site search bar in action, look no further than the site header. </p>
+
+<h2>Code</h2>
+{% highlight text %}
+{% include_relative includes/sitesearch.html %}
```

Comment:
We didn't got the proper label. For example the heading level 2 is empty. 

![image](https://github.com/wet-boew/GCWeb/assets/1676030/1749644d-d76b-4a36-9e62-68409a080bff)


We will need to include our variable in this page to get it work. For example this code would need to be added like before the "highlight text" jekyll function

```
{%- include variable-core.liquid -%}
```